tube_x: [1.4, 0.5, 0.76, -3, -2]
tube_y: [-0.75, 1.64, -2, -3.4, 1.75]
tube_radius: 0.0762
tube_var: [0.00001, 0., 0., 0.00001]
slip_min: 0.9
slip_max: 1
trans_var: 0.00001
rot_var: 0.00001
max_range: 3.5
min_range: 0.12
angle_inc: 0.01745329238474369
samples: 360
resolution: 0.015
laser_noise: 0.01
border_width: 10
border_height: 10
